library/core/src/cell.rs

```diff
@@ -814,34 +793,15 @@ impl Display for BorrowError {
 /// An error returned by [`RefCell::try_borrow_mut`].
 #[stable(feature = "try_borrow", since = "1.13.0")]
 #[non_exhaustive]
-<<<<<<< HEAD
 #[cfg(not(feature = "ferrocene_certified"))]
-=======
 #[derive(Debug)]
```

Comment:
you mentioned something about leaving the `cfg(certified)` as the last attribute all the time. Should we do that here?